lib/internal/process/permission.js

```diff
@@ -33,4 +33,15 @@ module.exports = ObjectFreeze({
 
     return permission.has(scope, reference);
   },
+  availableFlags() {
+    return [
+      '--allow-fs-read',
+      '--allow-fs-write',
+      '--allow-addons',
+      '--allow-child-process',
+      '--allow-net',
+      '--allow-wasi',
+      '--allow-worker',
```

Comment:
nit: If sequence doesn't matter, I think it would be better (grokability) to alphabetise them.